ai_agents/agents/examples/speechmatics-diarization/tenapp/ten_packages/extension/main_python/extension.py

```
@@ -31,6 +33,28 @@ class MainControlExtension(AsyncExtension):
     Consumes semantic AgentEvents from the Agent class and drives the runtime behavior.
     """
 
+    PLAYER_ALIAS_MAP: dict[str, list[str]] = {
+        "Elliot": ["elliot", "elliott", "elyot"],
+        "Musk": ["musk", "elon", "mass", "mask"],
+        "Taytay": ["taytay", "tay tay", "tate", "taylor", "swift", "tay"],
+    }
```

Ruff (0.14.0)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)